README.md

```diff
@@ -182,4 +180,4 @@ This project is inspired and helped by the following projects:
 * [adamyy/checkerframework-lsp](https://github.com/adamyy/checkerframework-lsp)
 * [georgewfraser/vscode-javac](https://github.com/georgewfraser/vscode-javac)
 * [adamvoss/vscode-languageserver-java-example](https://github.com/adamvoss/vscode-languageserver-java-example)
```

Comment:
Not related to this line: let's not commit the binary file `checker-framework-language-server-0.2.0.vsix`. It's the build output and doesn't need to be added to the repo.

According to https://nodejs.dev/en/learn/the-package-lock-json-file the `package-lock.json` file should be added to the repo, so that is fine.